### ISSUE TITLE
No vertical scrolling possible

### ISSUE THREAD

**Issue by kvonspiczak** [stevenwanderski/bxslider-4#1235]
i think i've found a regression. i just tried upgrading from 4.2.12 to 4.2.15 but the bug is still present.

see: https://github.com/stevenwanderski/bxslider-4/issues/82 & https://github.com/stevenwanderski/bxslider-4/issues/616

i cant scroll past the slider on a mobile device since it considers my scrolling as a "slide" event.

**Comment by Masner** [comment]
Bug is caused by preventDefault in onTouchStart function. I dont know, why is it there, so take it only as a hint.

**Comment by kvonspiczak** [comment]
@Masner yes you're right. i fixed it by patching the bxslider.js file in our project by the following changes:
![image](https://user-images.githubusercontent.com/2630690/53102310-af27d580-352b-11e9-90e8-212f94c653f5.png)
![image](https://user-images.githubusercontent.com/2630690/53102343-bd75f180-352b-11e9-8a2d-5d45d8a20915.png)
![image](https://user-images.githubusercontent.com/2630690/53102370-c8c91d00-352b-11e9-8972-19315a5b3d90.png)

**Comment by UngarMax** [comment]
> @Masner yes you're right. i fixed it by patching the bxslider.js file in our project with the following changes:
> ![image](https://user-images.githubusercontent.com/2630690/53102310-af27d580-352b-11e9-90e8-212f94c653f5.png)
> ![image](https://user-images.githubusercontent.com/2630690/53102343-bd75f180-352b-11e9-8a2d-5d45d8a20915.png)
> ![image](https://user-images.githubusercontent.com/2630690/53102370-c8c91d00-352b-11e9-8972-19315a5b3d90.png)
> 
> using version 4.2.15 (4.2.1d)

Thank you! That did the trick.

**Comment by giudansky** [comment]
lost days finding this solution.
So we can't use any cdn right? No official fixes or branch with this mod?

Thanks

**Comment by vorlox** [comment]
Thanks for the fix! You saved my day.

**Comment by jhume** [comment]
Same issue here.  Would be good to have an official fix for this rather than having to hack the codebase.

**Comment by giudansky** [comment]
any reason this fix was not submitted to the final ver ?
